src/main/java/org/jgroups/protocols/kubernetes/stream/CertificateStreamProvider.java

```diff
@@ -115,15 +116,18 @@ public InputStream openStream(String url, Map<String, String> headers, int conne
                 trustManagerFactory.init(trustStore);
 
                 return trustManagerFactory.getTrustManagers();
+            } catch (FileNotFoundException ignore) {
+                log.log(Level.WARNING, "ca cert file not found " + caCertFile + " - defaulting to insecure trust manager");
+                return TrustManagers.INSECURE_TRUST_MANAGERS;
             } catch (Exception e) {
                 log.log(Level.SEVERE, "Could not create trust manager for " + caCertFile, e);
```

Comment:
I think it would be better to use the unsecured trust manager here and log an error.

Reply:
@slaskawi This would happen when the store itself is invalid or misformatted, which should be very unlikely so failing here seems to me to be desirable.